dataframe/dataframe.go

```diff
@@ -122,7 +124,7 @@ func (df DataFrame) print(
 	}
 
 	if df.Err != nil {
-		str = fmt.Sprintf("%s error: %v", class, df.Err)
+		str = createErr("df.String()", "%s error: %v", class, df.Err).Error()
```

Comment:
Just noticed you do this in a couple of locations... You first create a wrapped error and then directly call `.Error()` to get a `string`. That doesn't make much sense right? Why not just create the `string` instead?